frontend/src/metabase/common/components/DataPicker/components/TablePicker.module.css

```diff
@@ -0,0 +1,3 @@
+.inner {
+  padding: 10dvh 10dvw !important;
```

Comment:
I don't like using `!important` tbh @ranquild seems it's going to be a common pattern

Reply:
Replied

Reply:
Short answer - don't add hacks